src/components/modules/ContributionsEpic.tsx

```diff
@@ -168,7 +147,18 @@ export const ContributionsEpic: React.FC<EpicProps> = ({
     onReminderOpen,
 }: EpicProps) => {
     const [isReminderActive, setIsReminderActive] = useState(false);
-    const { heading, backgroundImageUrl, showReminderFields, tickerSettings } = variant;
+    const { backgroundImageUrl, showReminderFields, tickerSettings } = variant;
```

Comment:
One general thing to mention - these are all changes in the `modules` version of the epic component. Do we also want these in the non `modules` version? Hopefully that one will go away soon, once we've entirely switched over the the dynamic import approach, but perhaps we should support this in both in the meantime?

Reply:
yes it should apply to any epics, is the template replacement still broken for some users?

Reply:
No, the incorrect placeholder was fixed for both in #173. I think the issue is that we don't yet have the final check for unreplaced placeholders in both versions yet.